pkg/cmd/cli/secrets/new.go

```diff
@@ -18,7 +19,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const NewSecretRecommendedCommandName = "new"
+const (
+	NewSecretRecommendedCommandName = "new"
+
+	newLong = `Create a new secret based on a key file or on files within a directory.
+
+	Key files can be specified using their file path, in which case a default name will be given to them, or optionally 
+	with a name and file path, in which case the given name will be used. Specifying a directory will create a secret 
+	using with all valid keys in that directory.
+`
+
+	newExamples = `  // Create a new secret named my-secret with a key named ssh-privatekey
+  $ %[1]s my-secret ~/.ssh/ssh-privatekey
+
+  // Create a new secret named my-secret with keys named ssh-privatekey and ssh-publickey, overwriting the name of the keys on disk
+  $ %[1]s my-secret ssh-privatekey=~/.ssh/id_rsa ssh-publickey=~/.ssh/id_rsa.pub
+
+  // Create a new secret named my-secret with keys foo and bar, by specifying the source directory that contains them
+  $ %[1]s my-secret path/to/foo/and/bar
```

Comment:
@liggitt do we want to expose that a directory containing anything but valid key names will cause a failure?